README.md

```diff
@@ -16,6 +16,25 @@ To get `help` command, run the following code:
 
 ## Commands
 
+To create and close milestone:
+```
+bot create milestone title myorg/myrepo
+bot close milestone title myorg/myrepo
+```
+To create and close all milestones listed in `config.json`:
+```
+bot create milestone title 
+bot close milestone title
+```
+`title` - should be in the following format `DD MMM YYYY`. For example: 13 Jan 2017
```

Comment:
Why should Title follow exactly this format? What if I want to create milestone called "backlog" or "icebox" etc...

Reply:
@zelima Good points. I forgot about the `backlog` and `icebox`. I will refactor it

Reply:
@zelima Title I am using is for vaildation, if someone uses `13 XXX 2017` it will throw an error `Invalid format` and, plus I is our convention and i will generate due_date from the title using `moment` library